omod/src/main/webapp/resources/htmlforms/Walkin-Client-Data-Capture-tool.xml

```diff
@@ -0,0 +1,243 @@
+                                        </span>
+                                        <obs id="eid-number" labelText="EID Number" conceptId="166453" />
+
+                                        <obs id="client-category" conceptId="165169" labelText="Client Category" answerConceptIds="90315, 165132, 159674, 162198, 165125, 90041, 99601, 157351, 99087,165133, 159991, 165127, 166461, 166462, 166463" answerLabels="MW- Migrant Workers, AGYW- Adoloscent Girls and Young Women, FF- Fisher Folk, LDT- Long Distance Truck Drivers, UF- Uniformed Forces, Preg- Pregnant, BF- Breastfeeding, IDU- Injectable Drug User, HEI- HIV Exposed Infant, OKP- Other KPs, TB- TB/HIV Client, RF- Refugee, ART- General ART Client, NIDU- Non IDU, OPP- Other PPs" />
+                                        <label>National ID Number</label>
+                                        <span id="nin-number">
+                                            <patient field="identifier" identifierTypeId="f0c16a6d-dc5f-4118-a803-616d0075d282" required="false" />
+                                        </span>
+                                    </div>
+                                </div>
+                            </div>
+                        </div>
+
+                        <div class="card">
+                            <div class="card-header">
+                                Vitals
+                            </div>
+                            <div class="card-body">                        
+                                <div class="row">
+                                    <div class="col-md-3">
+                                        <label>Weight</label>
+                                        <obs conceptId="5089" />
+                                        <label>Blood Pressure</label>
+                                        <div class="row ml-1">
+                                            <obs conceptId="5085" id="bp_systolic" />
+                                            <p class="left">
+                                                /
+                                            </p>
+                                            <obs conceptId="5086" id="bp_diastolic" />
+                                        </div>
+                                        <label>Temperature (c)</label>
+                                        <obs conceptId="CIEL:5088" id="temperature" />
+                                        
+                                    </div>
+                                </div>
+                                
+                            </div>
+                        </div>
+
+                        <div class="card">
+                            <div class="card-header">
+                                Clinical Assessment and Refill
+                            </div>
+                            <div class="card-body">
```

Comment:
@ODORA0 Change the order to match the PDF version of the register that I have attached to the Asana card today Aug 5, 2021

Reply:
@ssmusoke Done